Assets/Scripts/EnemyController.cs

```diff
@@ -14,22 +14,35 @@ public class EnemyController : MonoBehaviour
     private GameObject projectile;
     [SerializeField]
     private Transform player;
+    private Rigidbody2D rb2D;
+    private Vector2 targetPosition;
 
     float nextFire;
 
     // Start is called before the first frame update
     void Start()
     {
+        
         currentHealth = maxHealth;
         nextFire = Time.time;
     }
 
+    private void Awake() {
```

Comment:
There's no need to separate `Awake` and `Start` here. I think the code that's in `Awake` and `Start` in this script can all just go in the `Awake` method, but it doesn't really matter to me.